catalog/task/update-infra-deployments/0.1/README.md

```diff
@@ -0,0 +1,20 @@
+# update-infra-deployments
+
+* This task clones redhat-appstudio/infra-deployments GH repository.
+* It then runs a script obtained from the infra-deployment-update-script key in extraData which can modify text files.
+* It then generates pull-request for redhat-appstudio/infra-deployments repository using the modified files.
+
+
+## Parameters
+| Name                    | Description                                                                                  | Required | Default Value                                                                                                                                    |
```

Comment:
We use `Optional` instead of `Required` for our tasks and pipelines

Reply:
fixed

Reply:
Did you forget to git push?

Reply:
just did it now. sorry